internal/apiserver/clusterservice/clusterimpl.go

```diff
@@ -1322,6 +1322,12 @@ func getClusterParams(request *msgs.CreateClusterRequest, name string, userLabel
 	if request.BackrestS3Region != "" {
 		spec.BackrestS3Region = request.BackrestS3Region
 	}
+	if request.BackrestS3URIStyle != "" {
+		spec.BackrestS3URIStyle = request.BackrestS3URIStyle
+	}
```

Comment:
It would be good to do some validation of the `BackrestS3URIStyle` provided, e.g. to verify either `host` or `path` was specified.

Reply:
Error handling has been added, thanks for the review.